### ISSUE TITLE
Should be using chain_name instead of chain_id

### ISSUE THREAD

**Issue by sunnya97** [osmosis-labs/contract-list#2]

**Comment by ValarDragon** [comment]
Don't Keplr and cosmosstation index things by chain_id though?

**Comment by sunnya97** [comment]
Yeah, but you can map from chain_name -> chain_id using chain registry.  chain.jsons are indexed by chainname too.

If the chain id updates due to upgrade, the contract list should still be valid
